### PR TITLE
[3.5-dev] Fix some codestyle for the current 3.5-dev changes.

### PR DESCRIPTION
**Pull request by zero-24**
This PR fix some very minor CS Issues with the new plugin and the router changes.

There is absolutly no production code changed So there is no real testing possible :smile:

### CODE REVIEW

File: libraries/cms/component/router/view.php [hunk 1]
```diff
@@ -12,42 +12,42 @@
...
  */
 abstract class JComponentRouterView extends JComponentRouterBase
 {
 	/**
 	 * Name of the router of the component
 	 *
-	 * @var string
-	 * @since 3.4
+	 * @var   string
+	 * @since 3.5
```
Comment (mbabker) [inline libraries/cms/component/router/view.php]: Should be two spaces after the last tag

Comment (mbabker) [inline libraries/cms/component/router/view.php]: Err, longest, not last :-)

File: plugins/system/updatenotification/updatenotification.php [hunk 2]
```diff
@@ -142,11 +144,11 @@ public function onAfterRender()
 		}
 
 		// If we're here, we have updates. First, get a link to the Joomla! Update component.
-		$baseURL = JURI::base();
-		$baseURL = rtrim($baseURL, '/');
+		$baseURL  = JURI::base();
```
Comment (mbabker) [inline plugins/system/updatenotification/updatenotification.php]: Since the line is touched, can we change this to the correct `JUri::base()`?

**Comment by zero-24** [comment]
Thanks @mbabker should be fine now.

**Pull request merged**